cmd/error.go

```diff
@@ -0,0 +1,76 @@
+package cmd
+
+import (
+	"encoding/json"
+	"errors"
+)
+
+type Class int
+
+//go:generate enumer -type Class -trimprefix Class -transform=lower -text
+const (
+	_ Class = iota
+	ClassMeter
+	ClassCharger
+	ClassVehicle
+	ClassTariff
+	ClassCircuit
+	ClassSite
+	ClassMqtt
+	ClassDatabase
+	ClassEEBus
+	ClassJavascript
+	ClassGo
+	ClassHEMS
+	ClassInflux
+	ClassMessenger
```

Comment:
No ModbusProxy?

Reply:
Dazu gibts ein Todo. Kann egtl. fast keinen Fehler machen ausser Port belegt. Kannnich ergänzen.

Reply:
In der UI reagiere ich bereits auf `modbusproxy`. Brauchst also nur die Implementierung dafür ergänzen und dann sollte es laufen.

Reply:
Done, Sponsortoken ebenfalls hinzu gefügt!